### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
## In this PR:
-⭐️Use the provided template.
- ⭐️Create a database with the name: vet_clinic
- ⭐️Create a table with the name: animals
image

 -⭐️Insert the provided data
image

- ⭐️Queries:
 - Find all animals whose name ends with "mon"
image

 - List the names of all animals born between 2016 and 2019
image

 - List the name of all animals that are neutered and have less than 3 escape attempts
image

 - List the date of birth of all animals named either "Agumon" or "Pikachu"
image

 - List name and escape attempts of animals that weigh more than 10.5kg
image

 - Find all animals that are neutered
image

 - Find all animals not named Gabumon
image

 - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equal precisely 10.4kg or 17.3kg)
 
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,35 +134,25 @@ Feel free to check the [issues page](../../issues/).
...
-
-  - [Answer_1]
-
-- **[Question_2]**
+- **[Can I use this project for my website?]**
 
-  - [Answer_2]
+  - [Sure It's MIT license, feel free to use as you like]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (AbayomiOlaoye) [inline README.md:157]: - [x] It appears you do not have a `License` file in your directory which you were trying to link to the license to. It's currently showing an error 404 page :point_down: 
image

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @MostafaShahat2021 :wave: :wave:, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so_close


 ## Highlights
- Good PR message ✅ 
- Great commit messages ✅
- Project requirements met :heavy_check_mark: 
- No linter error ✅
- Approved README template :heavy_check_mark: 

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (MostafaShahat2021) [reply README.md:157]: Hi @AbayomiOlaoye,
I hope all is well with you 👍 ,
thanks a lot for your efforts in reviewing my code,  and for your suggestion, I'll fix that 😄
Best Regards ✌️

**Review by usorfaitheloho — APPROVED** [review]
Hi @MostafaShahat2021  , 
# STATUS: APPROVED! 🥇 💯 

done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hi @usorfaitheloho, Thanks a lot 💯 for your efforts in reviewing my code 😄 .

**Pull request merged**